dnf5/commands/history/history_store.cpp

```diff
@@ -36,11 +36,11 @@ void HistoryStoreCommand::set_argument_parser() {
     auto & parser = ctx.get_argument_parser();
 
     output_option = dynamic_cast<libdnf5::OptionString *>(
-        parser.add_init_value(std::make_unique<libdnf5::OptionString>("./transaction.json")));
+        parser.add_init_value(std::make_unique<libdnf5::OptionString>("./transaction")));
     auto query_format = parser.add_new_named_arg("output");
     query_format->set_long_name("output");
     query_format->set_short_name('o');
-    query_format->set_description("File path for storing the transaction, default is \"./transaction.json\"");
+    query_format->set_description("Path to a directory for storing the transaction, default is \"./transaction\"");
```

Comment:
May I ask you why a directory and not a file path?

File path allow to store more transaction files in one directory. Directory might make sense when it would be used as drop in directory, but it is not. It requires to have it in file called `transaction.json`.

Reply:
With the current issue I discovered that passing a directory is very inconvenient.

Reply:
I am not sure I understand what you mean.
If we allowed setting arbitrary file name for the json then `replay` wouldn't find it, it has hard-coded `transaction.json`.

Reply:
I am so sorry, I was confused here. The current implementation is good.